src/coreclr/tools/Common/TypeSystem/IL/MethodIL.cs

```diff
@@ -86,7 +86,7 @@ public abstract partial class MethodIL
         /// (typically a <see cref="MethodDesc"/>, <see cref="FieldDesc"/>, <see cref="TypeDesc"/>,
         /// or <see cref="MethodSignature"/>).
         /// </summary>
-        public abstract Object GetObject(int token, NotFoundBehavior notFoundBehavior = NotFoundBehavior.ReturnNull);
+        public abstract Object GetObject(int token, NotFoundBehavior notFoundBehavior = NotFoundBehavior.Throw);
```

Comment:
This is inconsistent with the default for `EcmaMethodIL.GetObject` and with the original behavior.